common/metrics/metricstest/metricstest.go

```diff
@@ -193,10 +244,52 @@ func (s Snapshot) MustGauge(name string, tags ...metrics.Tag) float64 {
 	return v
 }
 
+func (s Snapshot) Histogram(name string, tags ...metrics.Tag) ([]HistogramBucket, error) {
+	labelValues := map[string]string{}
+	for _, tag := range tags {
+		labelValues[tag.Key()] = tag.Value()
+	}
+
+	sample, ok := s.histogramSamples[name]
+	if !ok {
+		return nil, fmt.Errorf("metric %s not found", name)
+	}
```

Comment:
I don't think this should be an error. I think we should return an empty list--that's how PromQL would behave

Reply:
I see. Ok, yeah that makes sense